aogolfevent.civix.php

```diff
@@ -450,16 +470,14 @@ function _aogolfevent_civix_civicrm_alterSettingsFolders(&$metaDataFolders = NUL
  *
  * Find any *.entityType.php files, merge their content, and return.
  *
- * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_entityTypes
+ * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_entityTypes
  */
-
 function _aogolfevent_civix_civicrm_entityTypes(&$entityTypes) {
-  $entityTypes = array_merge($entityTypes, array (
-    'CRM_Aogolfevent_DAO_EventContributionPF' =>
-    array (
+  $entityTypes = array_merge($entityTypes, [
+    'CRM_Aogolfevent_DAO_EventContributionPF' => [
       'name' => 'EventContributionPF',
       'class' => 'CRM_Aogolfevent_DAO_EventContributionPF',
-      'table' => 'civicrm_event_contribution_pf',
-    ),
-  ));
+      'table' => 'civicrm_event_contribution_p_f',
```

Comment:
O_o this looks like to be a potential problem here @monishdeb

Reply:
nice catch :) .. updated now.